### PR TITLE
Time graph

### PR DESCRIPTION
**Pull request by vincenzobaz**
Make first use of timeSpent in a simple bar chart.

Please wait until stats is fixed to test and merge this PR.

### CODE REVIEW

**Comment by vincenzobaz** [comment]
Now testing

File: meteor/imports/client/components/stats/TimePerTypeBarChart.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {Stats} from "../../../common/models/Stats";
+import {Kinds} from "../../../common/models/questions/common/Kind";
+const Bar = require('react-chartjs-2').Bar;
+
+
+export function TimePerTypeBarChart(props) {
+   const stats: Stats[] = props.stats;
+
+   let timePerType: {[key: string]: number} = {};
```
Comment (Rabyss) [inline meteor/imports/client/components/stats/TimePerTypeBarChart.tsx]: Add:
```javascript
let amountPetType: {[key: string]: number} = {};
```

File: meteor/imports/client/components/stats/TimePerTypeBarChart.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   Kinds.forEach(kind => timePerType[kind] = 0);
```
Comment (Rabyss) [inline meteor/imports/client/components/stats/TimePerTypeBarChart.tsx]: Add:
```javascript
Kinds.forEach(kind => amountPerType[kind] = 0);
```

File: meteor/imports/client/components/stats/TimePerTypeBarChart.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+   for (let stat of stats) {
+       for (let kind of Kinds) {
+           timePerType[kind] += stat.counters[kind].timeSpent;
```
Comment (Rabyss) [inline meteor/imports/client/components/stats/TimePerTypeBarChart.tsx]: Add:
```javascript
amountPerType[kind] += stat.counters[kind].amount;
```

File: meteor/imports/client/components/stats/TimePerTypeBarChart.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+       }
+   }
+
+   let averages: number[] = Kinds.map(kind => timePerType[kind] / stats.length);
```
Comment (Rabyss) [inline meteor/imports/client/components/stats/TimePerTypeBarChart.tsx]: Repalce by:
```javascript
let averages: number[] = Kinds.map(kind => timePerType[kind] / amountPerType[kind]);
```

**Pull request closed without merging**